packages/react-composites/src/composites/ChatComposite/adapter/AzureCommunicationChatAdapter.ts

```diff
@@ -29,9 +29,13 @@ import {
   TopicChangedListener
 } from './ChatAdapter';
 import { AdapterError } from '../../common/adapters';
+/* @conditional-compile-remove-from(stable): FILE_SHARING */
+import { FileSharingAdapter } from './FileSharingAdapter';
+/* @conditional-compile-remove-from(stable): FILE_SHARING */
+import { ObservableFileUpload } from '../file-sharing';
 
 /** Context of Chat, which is a centralized context for all state updates */
```

Comment:
Can you mark this `@private` so that  it's not accidentally exported in the future.